mate-panel/libmate-panel-applet-private/panel-applet-container.c

```diff
@@ -820,17 +798,17 @@ mate_panel_applet_container_child_popup_menu (MatePanelAppletContainer *containe
 					 GAsyncReadyCallback   callback,
 					 gpointer              user_data)
 {
-	GSimpleAsyncResult *result;
+	GTask              *task;
 	GDBusProxy         *proxy = container->priv->applet_proxy;
 
 	if (!proxy)
 		return;
-
-	result = g_simple_async_result_new (G_OBJECT (container),
-					    callback,
-					    user_data,
-					    mate_panel_applet_container_child_popup_menu);
-
+	task = g_task_new (G_OBJECT (container),
+	                                 cancellable,
+	                                 callback,
+	                                 user_data);
+	g_task_set_source_tag (task,mate_panel_applet_container_child_popup_menu);
+	
```

Comment:
Missing tabs and wrong alignment. And no tab in beginning of line 811 please. Should be
```
-	result = g_simple_async_result_new (G_OBJECT (container),
-					    callback,
-					    user_data,
-					    mate_panel_applet_container_child_popup_menu);
-
+	task = g_task_new (G_OBJECT (container),
+			   cancellable,
+			   callback,
+			   user_data);
+	g_task_set_source_tag (task,mate_panel_applet_container_child_popup_menu);
+
```